doc/rst/conf.py

```diff
@@ -117,7 +117,8 @@ def setup(app):
                     'builtins/ChapelArray.rst',
                     'builtins/ChapelDomain.rst',
                     'builtins/ChapelLocale.rst',
-                    'builtins/ChapelRange.rst'
+                    'builtins/ChapelRange.rst',
+                    'builtins/Bytes.rst'
```

Comment:
This list looks alphabetized, so I think this one should go before ChapelArray (it'll also maybe help you avoid merge conflicts with the string PR?)